src/org/zaproxy/zap/view/ContextListTableModel.java

```diff
@@ -40,9 +40,6 @@
     
     //private static Logger log = Logger.getLogger(ContextListTableModel.class);

```

Comment:
Not specifically related,  but could go too? (With a minor tweak to the commit msg.)